common/stacks/src/main/scala/weco/api/stacks/http/SierraSource.scala

```diff
@@ -31,33 +37,67 @@ class SierraSource(client: HttpClient with HttpGet with HttpPost)(
   private implicit val umItemStub: Unmarshaller[HttpEntity, SierraItemData] =
     CirceMarshalling.fromDecoder[SierraItemData]
 
+  private implicit val umItemEntriesStub
+    : Unmarshaller[HttpEntity, SierraItemDataEntries] =
+    CirceMarshalling.fromDecoder[SierraItemDataEntries]
+
   private implicit val umErrorCode: Unmarshaller[HttpEntity, SierraErrorCode] =
     CirceMarshalling.fromDecoder[SierraErrorCode]
 
+  /** Returns data for a list of items
+    *
+    * Note: if some of the IDs requested do not exist, they will
+    * not appear in the results list. It's up to the consumer to
+    * identify that and react to it as required.
+    */
+  def lookupItemEntries(
+    items: Seq[SierraItemNumber]
+  ): Future[Either[SierraItemLookupError, SierraItemDataEntries]] = {
+
+    val idList = items
+      .map(_.withoutCheckDigit)
+      .mkString(",")
+
+    for {
+      response <- client.get(
+        path = Path("v5/items"),
+        params = Map(
+          "id" -> idList,
```

Comment:
Is there a limit to how many items we can fetch in a single response? There's a work with 500+ items.

Reply:
Probably. The API documentation doesn't specify a limit AFAICT but I bet there is one. I'm not going to account for that in this PR but will test and update in the future if required.